cmd/api/src/database/migration/migrations/v5.16.0.sql

```diff
@@ -14,11 +14,13 @@
 --
 -- SPDX-License-Identifier: Apache-2.0
 
+-- Add Citrix RDP
+INSERT INTO parameters (key, name, description, value, created_at, updated_at) VALUES ('analysis.citrix_rdp_support', 'Citrix RDP Support', 'This configuration parameter toggles Citrix support during post-processing. When on, CanRDP edges will come from the `Direct Access Users` group instead of the builtin `Remote Desktop Users` group.', '{"enabled": false}', current_timestamp, current_timestamp) ON CONFLICT DO NOTHING;
```

Comment:
The description here should be updated with the new copy: `This configuration parameter toggles Citrix support during post-processing. When enabled, computers identified with a 'Direct Access Users' local group will assume that Citrix is installed and CanRDP edges will require membership of both 'Direct Access Users' and 'Remote Desktop Users' local groups on the computer.`

Reply:
This will be addressed in https://github.com/SpecterOps/BloodHound/pull/818 as this branch is on top of that branch.